torch/distributed/tensor/_ops/_pointwise_ops.py

```diff
@@ -631,10 +641,25 @@ def common_pointwise_strategy(
     return pointwise_strategy
 
 
+redistribute_partial_ops = [aten.add.Tensor, aten.add_.Tensor]
+
+
+norm_partial_avoidable_redistribute_ops = [
+    aten.div.Scalar,
+    aten.div_.Scalar,
+    aten.mul.Scalar,
+    aten.mul_.Scalar,
+]
+
 for op in linear_pointwise_ops:
-    register_op_strategy(op, schema_info=RuntimeSchemaInfo(static_kwargkey=["out"]))(
-        linear_pointwise_strategy
-    )
+    if op in norm_partial_avoidable_redistribute_ops:
+        register_op_strategy(
+            op, schema_info=RuntimeSchemaInfo(1, static_kwargkey=["out"])
```

Comment:
remind me what the meaning of this `1` is for caching? and why it relates to NormPartial